### ISSUE TITLE
Images dupplication in posts

### ISSUE THREAD

**Issue by Gilbertdelyon** [humhub/humhub#7292]
See also https://github.com/humhub/humhub/issues/7288#issue-2626448957. 
This new issue is posted after deeper investigation

### What steps will reproduce the problem?
**Fresh HH1.16.2 - Native theme - Only "Messenger" and "Updater" Modules**
**Step1:** 
Create a post 
Add an IMAGE  via "Attach an image" editor's menu item
Publish the post
So far so good

**Step2**
Create a new post
Add the SAME IMAGE via "Attach an image" editor's menu item
Publish the post
**Not good 👎** - The image is displayed **2 times** in this post.

**Step3**
Create a new post
Add the SAME IMAGE via "Attach an image" editor's menu item
Publish the post
**Not good 👎** - The image is displayed **3 times** in this post.
etc...

### What is the expected result?
Images should not be dupplicated

### Additional info
- This issue will not occur if you  delete the browser cache before adding a post
- If you delete the first post before posting the second one, the image will be also dupplicated
- Same issue in Chrome and FFox

**Comment by marc-farre** [comment]
@Gilbertdelyon Thanks for the detailed steps.
The issue has been confirmed by HumHub team.

**Comment by yurabakhtin** [comment]
@Gilbertdelyon I could reproduce the bug on fresh HH1.16.2 and if run `composer install` because I see only version `2.1.3` of the package `npm-asset/humhub-prosemirror-richtext` is used there.
But after run `composer update` I see the package is updated to `2.1.4` and the version has no the bug.
I guess the bug was fixed in the PR https://github.com/humhub/humhub-prosemirror/pull/147.
Could you please try to update the composer package and test again?
Thanks.

**Comment by Gilbertdelyon** [comment]
@yurabakhtin 
1/ My test site is HH1.16.2. It was built by updating a fresh installation HH1.16.0 that I successively updated to 1.16.1 and 1.16.2
2/ This is what I see in hhsite_root_dir/protected/vendor/composer/installed.json
```
{
            "name": "npm-asset/humhub-prosemirror-richtext",
            "version": "2.1.3",
            "version_normalized": "2.1.3.0",
            "dist": {
                "type": "tar",
                "url": "https://registry.npmjs.org/humhub-prosemirror-richtext/-/humhub-prosemirror-richtext-2.1.3.tgz"
            },
            "type": "npm-asset",
            "installation-source": "dist",
            "license": [
                "Apache-2.0"
            ],
            "install-path": "../npm-asset/humhub-prosemirror-richtext"
        },
```
3/ Sorry I am not very familiar with "Composer" refinements. I use it as  a  cook receipe for my "Drupal" site updating but I never used it in HH.
Would you please detail the complete command line I need to run,
 example:
`[my_server subdirectory_where_I should_locate]$ composer update/someting --with-all-dependencies`

**Comment by Gilbertdelyon** [comment]
This is what I did
and this is what I got:
```
Composer could not find a composer.json file in /home/blabla/hh_root_site_dir
To initialize a project, please create a composer.json file. See https://getcomposer.org/basic-usage
```

**Comment by yurabakhtin** [comment]
@Gilbertdelyon If I download the file "Source code (zip)" from [here](https://github.com/humhub/humhub/releases/tag/v1.16.2), I can find the file `composer.json` in the root of the zip archive.

![root](https://github.com/user-attachments/assets/4c455ec8-6ba6-4d09-a0b0-01027d000e60)

> Should I copy/paste the `/protected/vendor/composer/installed.json` in the site root dir?

No, you shouldn't, it is a different.

**Comment by Gilbertdelyon** [comment]
Ahhh I see!!
So there is an issue because 
- as I wrote in previous comment _"The site was not initially installed with composer, but manually by decompressing the HH1.16.0zip file"_
- And I cannot see composer.json file neither in HH1.16.0 nor in my test site HH1.16.0 uptaded to HH1.6.1 -> uptated to HH1.16.2

So I understand that a fresh 1.16.2 files is not the same as an old site HH1.16.0 uptaded to HH1.6.1 -> uptated to HH1.16.2

So, what should I do?

**Comment by yurabakhtin** [comment]
> So, what should I do?

If you have a possibility to run `composer update` from the root folder(see on my screenshot to know how the root should look) then please do this, otherwise I don't know how to help you, only if you will wait a next release with new version of the `prosemirror` package.

**Comment by Gilbertdelyon** [comment]
This is what I did:
Copy `composer.json` from [here](https://github.com/humhub/humhub/releases/tag/v1.16.2),
Paste it in my HH1.16.2 test site
Run composer update in the root dir of the site
I dont know if it's good or bad practice but now I have the prosemirror 2.1.5 installed and no duplicate. Yesss. :-))

So far so good for the test site and main production site, but
I cannot run composer in 2 other sites. (no access to cPanel terminal)
Should I understand that I will get the duplicate images in these sites up to next HH release?

**Comment by yurabakhtin** [comment]
> I dont know if it's good or bad practice but now I have the prosemirror 2.1.5 installed and no duplicate. Yesss. :-))

Yes, it is good, the latest version is [2.1.5](https://github.com/humhub/humhub-prosemirror/releases/tag/v2.1.5) now.

> Should I understand that I will get the duplicate images in these sites up to next HH release?

If the sites have old broken version 2.1.3 then unfortunately images will be duplicated.

> Shall we wait for 1.16.3 or 1.17.0 to get the last prosemirror package?

I think yes, the updating to new HH version should update the package `prosemirror` to new version 2.1.5.
